community/contributing/docs/reviewing-prs.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: single
+title: Reviewing Pull Requests
+sidebar:
+  nav: community
+---
+
+{% include toc %}
+
+# Review a pull request
+
+```
+Anyone can review a documentation pull request. Visit the [pull requests](https://github.com/spinnaker/spinnaker.github.io/pulls) section in the Spinnaker website repository to see a list of all currently open pull requests.
+
+Reviewing documentation pull requests is a great way to get started contributing to the Spinnaker community. It helps you to learn the code base, and builds rapport with other contributors.
+
+Before reviewing code, do the following:
+```
```

Comment:
I don't think code formatting is what you're looking for here.

Reply:
Huh, not sure why that happened. Thank you for the catch, I've gone ahead and removed those! :)